src/tests/class_tests/openms/source/PeakIntegrator_test.cpp

```diff
@@ -73,7 +73,7 @@ vector<double> position = {
   3.190133333,3.198016667,3.205916667,3.213166667
 };
 
-vector<double> intensity = {
+const vector<double> intensity = {
```

Comment:
To reaffirm, the "ideal" peak case is also included in the test sets along with the saturated, cutoff, and saturated + cutoff sets?

Reply:
It currently is not. I should take care of that, thanks